### ISSUE TITLE
bug: unable to set CRON pm2 via cli

### ISSUE THREAD

**Issue by smashah** [open-wa/wa-automate-nodejs#2709]
The problem is in this logic:
https://github.com/open-wa/wa-automate-nodejs/blob/f431fee7a9f1c211330b40686565f140e403c5d0/bin/server.js#L18

**Comment by smashah** [comment]
@github-actions run

<details>
<summary>⚡ Release!  ⚡</summary>

```js
(async () => {
function exec(cmd) {
  console.log(execSync(cmd).toString());
}
//set the version type
process.env.VERS = "patch"

// Config
const gitUserEmail = "github-actions[bot]@users.noreply.github.com";
const gitUserName = "github-actions[bot]";

exec(`echo "//registry.npmjs.org/:_authToken=$NPM_TOKEN" > .npmrc`);
exec(`git config --global user.email "${gitUserEmail}"`);
exec(`git config --global user.name "${gitUserName}"`);
exec(`npm i -D`);
exec(`npm run release-ci $VERS`);

// types only package
exec('npm config set //registry.npmjs.org/:_authToken $NPM_TOKEN')
exec(`cd ./types-only && npm version $VERS && npm run build && npm publish && cd ..`);
exec(`git commit -a -m 'updated types-only package'`);
exec(`git push --force`);

//comment on the issue
var result = execSync(`npx auto-changelog -o ./tempchangelog.txt --commit-limit false --template ./compact-keepachangelog.hbs --stdout`).toString();

    await postComment(result);

//create changelog image
exec(`npm run release-image`);
exec(`git commit -a -m 'updated release-image'`);
exec(`git push --force`);
  })();
```
</details>

**Comment by smashah** [comment]
# Changelog

## 🚀 Release [4.36.2](https://github.com/open-wa/wa-automate-nodejs/compare/4.36.1...4.36.2) (2022-05-15)

- Release 4.36.2 [`c9632d8`](https://github.com/open-wa/wa-automate-nodejs/commit/c9632d864a512d15ceb4da315689cbbc8b65dc46)
- 📝 fix location button docs [`f431fee`](https://github.com/open-wa/wa-automate-nodejs/commit/f431fee7a9f1c211330b40686565f140e403c5d0)
- 🐛 fix: `--pm2` flag formatting issues #2709 [`5b494ae`](https://github.com/open-wa/wa-automate-nodejs/commit/5b494ae534a48a44a4af622e8aa34fdc39c48a19)
- 🥚 patches update [`9a2d457`](https://github.com/open-wa/wa-automate-nodejs/commit/9a2d457158f1ac16b6b09c06ef020cd8355ec8c1)
- 🥚 patches update [`0cc28be`](https://github.com/open-wa/wa-automate-nodejs/commit/0cc28bed6afc28e20fef3c79927e4836cdd96acc)
- 🥚 patches update [`f08664b`](https://github.com/open-wa/wa-automate-nodejs/commit/f08664b3392015f0120b2a7ac6d340e03dc69ed6)
- updated types-only package [`11f869d`](https://github.com/open-wa/wa-automate-nodejs/commit/11f869d19af1df7d3c78458424b0b0e39f116dac)
